### PR TITLE
Add new file icons to TreeView

### PR DESCRIPTION
**Pull request by AndreyKozhev**
Add new file icons to TreeView.
# Changes
+ Added icons for:
  + `.kts`
  + `gradlew`
  + `gradlew.bat`
  + `.properties`
  + `.apk`
  + `.txt`
  + `.log`
  + `.cpp`
  + `.h`
+ Updated icons for:
  + `.gradle`
# Screenshots
<p float="center">

<p/>
<p float="center">

</p>

### CODE REVIEW

**Comment by itsaky** [comment]
I think the `ic_terminal` icon could be used for `gradlew` and `gradlew.bat` files.

**Comment by AndreyKozhev** [comment]
Okay.

File: app/src/main/java/com/itsaky/androidide/adapters/viewholders/FileTreeViewHolder.java [hunk 1]
```diff
@@ -103,12 +103,13 @@ protected int getIconForFile(final File file) {
     else if (file.getName().endsWith(".java")) icon = R.drawable.ic_language_java;
     else if (file.getName().endsWith(".kt") || file.getName().endsWith(".kts")) icon = R.drawable.ic_language_kotlin;
     else if (file.getName().endsWith(".xml")) icon = R.drawable.ic_language_xml;
-    else if (file.getName().endsWith(".gradle") || file.getName().endsWith("gradlew") || file.getName().endsWith("gradlew.bat")) icon = R.drawable.ic_language_gradle;
+    else if (file.getName().endsWith(".gradle")file.getName().endsWith("gradlew") || file.getName().endsWith("gradlew.bat")) icon = R.drawable.ic_language_gradle;
```
Comment (itsaky) [inline app/src/main/java/com/itsaky/androidide/adapters/viewholders/FileTreeViewHolder.java:106]: Isn't this a syntax error?

**Review by itsaky — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/itsaky/androidide/adapters/viewholders/FileTreeViewHolder.java (hunk 1 above)
Comment (AndreyKozhev) [reply app/src/main/java/com/itsaky/androidide/adapters/viewholders/FileTreeViewHolder.java]: Ready

**Review by itsaky — APPROVED** [review]

**Pull request merged**